README.md

```diff
@@ -1 +1,85 @@
 # draft-store-client
+
+[![Greenkeeper badge](https://badges.greenkeeper.io/hmcts/cmc-citizen-frontend.svg)](https://greenkeeper.io/)
+
+[![Standard - JavaScript Style Guide](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
+
+This is the draft store client module to access draft store. 
```

Comment:
This is a client library for interacting with the [draft store api](link to repo)

Reply:
updated